spec/support/shared_examples/abilities/agency_examples.rb

```diff
@@ -16,6 +16,34 @@
     should be_able_to(:review_convictions, WasteCarriersEngine::RenewingRegistration)
   end
 
+  context ":write_off_small" do
+    let(:finance_details) { build(:finance_details, balance: balance) }
+
+    context "when the zero difference balance is less than the setted cap" do
```

Comment:
```suggestion
    context "when the zero-difference balance is less than the write-off cap for agency users" do
```

[NIT] Maybe personal preference, but I think this is clearer about what the cap is. The rewording would apply to some other specs as well.